htdocs/core/class/html.form.class.php

```diff
@@ -5342,7 +5342,7 @@ function selectForFormsList($objecttmp, $htmlname, $preselectedvalue, $showempty
 					foreach($tmparray as $key => $val)
 					{
 						$val = preg_replace('/t\./','',$val);
-						$label .= (($label && $obj->$val)?' - ':'').$obj->$val;
+						$label .= (($label && $obj->$val)?' - ':'').($obj->$val != $langs->trans($obj->$val)? $langs->trans($obj->$val) : $obj->$val);
```

Comment:
Trying to translate always is not a good idea. This select can be used to show list of thirdparty and if a thirdparty is called HELLO (for HELLO company), then it will show the company Bonjour.
We must add a parameter in the function to say "try to translate value" or not.